client.go

```diff
@@ -169,7 +306,7 @@ func (cli *Client) Delete(ctx context.Context, port int, replay bool) (*Imposter
 // See more information about this resource at:
 // http://www.mbtest.org/docs/api/overview#delete-imposter-requests.
 func (cli *Client) DeleteRequests(ctx context.Context, port int) (*Imposter, error) {
-	p := fmt.Sprintf("/imposters/%d/requests", port)
+	p := fmt.Sprintf("/imposters/%d/savedProxyResponses", port)
```

Comment:
I updated the route of this call per the v1.16.0 update - see http://www.mbtest.org/feed.

Reply:
ratified against http://www.mbtest.org/releases/v1.16.0